pkg/ddevapp/traefik_config_template.yaml

```diff
@@ -2,6 +2,11 @@
 # If you remove the ddev-generated line above you
 # are responsible for maintaining this file. DDEV will not then
 # update it, for example if you add `additional_hostnames`, etc.
+# However, it is now unnecessary and imprudent to take control of
+# this file, as there is now a dynamic merge mechanism. To use it
+# create <project>/.ddev/traefik/dynamic_config.*.yaml files which
+# will then be merged in alphanumeric order. See the readme in the
+# /traefik directory for more information
```

Comment:
None of this is coming out in the resultant file, which just looks like this:
```
#ddev-generated
http:
    middlewares:
```

You'll need to prepend this string to the overall string before writing it out.

Reply:
This comment content is lost in the later merge, so this heading is irrrelevant, you can remove it. To see this, just step through the code and look at the string that results from the MergeYamlFiles(), it's gone.

Reply:
You already have enough in the final yaml don't you?